### PR TITLE
Assignment7

### PR DESCRIPTION
**Pull request by RichardTeller**
Note I attempted adding a page for adding/removing users from groups.

### CODE REVIEW

File: SecretSanta/src/SecretSanta.Web/Views/Users/Remove.cshtml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿@model UserViewModel
+@if (!string.IsNullOrWhiteSpace(ViewBag.ErrorMessage))
+{
+    <div>@ViewBag.ErrorMessage</div>
+}
+<form asp-route="Remove">
+    <div class="field">
+        <label class="label" asp-for="Id">User Id</label>
```
Comment (grantdwoods) [inline SecretSanta/src/SecretSanta.Web/Views/Users/Remove.cshtml]: Suggestions I made on the Users and Groups Index.cshtml files would save everyone the hassle of somehow knowing the Id's.

File: SecretSanta/src/SecretSanta.Web/Views/Users/Update.cshtml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿@model UserViewModel
+@if (!string.IsNullOrWhiteSpace(ViewBag.ErrorMessage))
+{
+    <div>@ViewBag.ErrorMessage</div>
+}
+    <form asp-route="Update">
+        <div class="field">
+            <label class="label" asp-for="Id">User Id</label>
```
Comment (grantdwoods) [inline SecretSanta/src/SecretSanta.Web/Views/Users/Update.cshtml]: Same comment as Users/Remove.cshtml.

File: SecretSanta/src/SecretSanta.Web/Controllers/GroupsController.cs [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Remove(int id)
+        {
+            IActionResult result = View();
+
+            if (ModelState.IsValid)
```
Comment (grantdwoods) [inline SecretSanta/src/SecretSanta.Web/Controllers/GroupsController.cs]: The ModelState check isn't really doing anything here, unless you were to:
-Check the int
-Decide it was invalid
-Set an error on the model state manually

If you were to pass a number that was <1 for instance, the ModelState.IsValid property would still be true

**Comment by breaman** [comment]
Nice work overall on the assignment, this looks good.

**Pull request closed without merging**